src/cli/commands/retrieve_history.go

```diff
@@ -33,6 +33,9 @@ func (command HistoryCommand) Execute([]string) error {
 		err    error
 		writer *os.File
 	)
+	if command.Desc && command.Asc {
+		return fmt.Errorf(ui.ConflictDisplayOrder)
```

Comment:
How about adding a "deprecation" notice when --desc is detected?

Reply:
then:
(1) for `cf asm xxx xxx --desc --asc`, return `Failed` and show the "deprecation" notice
(2) for `cf asm xxx xxx --desc`, return results and show the "deprecation" notice
is that right?

Reply:
yes, exactly.